spec/services/support_interface/ministerial_report_candidates_export_spec.rb

```diff
@@ -246,11 +253,9 @@
 
     context 'when the candidate has two choices with 2 pairs of matching subjects' do
       it 'correctly allocates the candidate' do
-        create_double_choice_application(
-          :with_accepted_offer,
-          %w[F0 08],
-          :with_declined_offer,
-          %w[F3 L1],
+        create_application(
+          CourseChoice.new('Thermodynamics', %w[F0 08], :with_accepted_offer),
```

Comment:
Silly course names here so that they deliberately don't match a known subject (Physics in this case).